regolith/interact.py

```diff
@@ -1,13 +1,14 @@
 """
 Loads the dbs for interactive sessions
 """
-from regolith.database import connect
-from regolith.runcontrol import DEFAULT_RC, load_rcfile, filter_databases
+from regolith.runcontrol import DEFAULT_RC, load_rcfile, filter_databases, \
+    connect_db
 
 rc = DEFAULT_RC
 rc._update(load_rcfile("regolithrc.json"))
 filter_databases(rc)
 
-with connect(rc) as rc.client:
-    dbs = rc.client.dbs
-    chained_db = rc.client.chained_db
+chained_db, dbs = connect_db(rc)
+
+
+
```

Comment:
Please only have a single newline at the end of the file

Reply:
ok, fixed in next push.